src/benchmarks/gc/README.md

```diff
@@ -429,7 +463,7 @@ This is the build output of coreclr that is used to run benchmarks.
 
 These are specified in the `coreclrs` section of a benchfile.
 
-This can be found in a directory like `bin/tests/Windows_NT.x64.Release/Tests/Core_Root` (adjust for different OS or architecture) of a coreclr repository. (The Core_Root can be moved anywhere and doesn't need to remain inside the coreclr repository.)
+This can be found in a directory like `bin/tests/Windows_NT.x64.Release/Tests/Core_Root` (adjust for different OS or architecture) of a coreclr repository. (The Core_Root can be moved anywhere and doesn't need to remain inside the coreclr repository).
 
 A clone of https://github.com/dotnet/coreclr,  which may be on an arbitrary commit (including one not checked in).
```

Comment:
Can you also update the docs to refer to the runtime repo instead of the CoreCLR repo?

Reply:
That's a change worth doing. At the time Andy wrote this documentation, runtime was not around yet.

Reply:
could you please do what @cshung suggested since coreclr repo is obsolete now?

Reply:
I have changed it now to runtime. I also built stuff on the runtime repo and updated this entire section.